stack-8.10.6.yaml

```diff
@@ -0,0 +1,98 @@
+  - ./plugins/hls-eval-plugin
+  - ./plugins/hls-explicit-imports-plugin
+  - ./plugins/hls-refine-imports-plugin
+  - ./plugins/hls-hlint-plugin
+  - ./plugins/hls-retrie-plugin
+  - ./plugins/hls-splice-plugin
+  - ./plugins/hls-tactics-plugin
+  - ./plugins/hls-brittany-plugin
+  - ./plugins/hls-stylish-haskell-plugin
+  - ./plugins/hls-floskell-plugin
+  - ./plugins/hls-fourmolu-plugin
+  - ./plugins/hls-pragmas-plugin
+  - ./plugins/hls-module-name-plugin
+  - ./plugins/hls-ormolu-plugin
+
+ghc-options:
+  "$everything": -haddock
+
+extra-deps:
+  - apply-refact-0.9.3.0
+  - brittany-0.13.1.2
+  - Cabal-3.0.2.0
+  - clock-0.7.2
+  - data-tree-print-0.1.0.2@rev:2
+  - floskell-0.10.5
+  - fourmolu-0.3.0.0
+  - # ghc-api-compat-8.6
+    github: hsyl20/ghc-api-compat
+    commit: 8fee87eac97a538dbe81ff1ab18cff10f2f9fa15
+  - ghc-check-0.5.0.4
+  - ghc-exactprint-0.6.4
+  - ghc-source-gen-0.4.1.0
+  - heapsize-0.3.0
+  - implicit-hie-cradle-0.3.0.5
+  - implicit-hie-0.1.2.6
+  - monad-dijkstra-0.1.1.2
+  - refinery-0.4.0.0
+  - retrie-1.0.0.0
+  - stylish-haskell-0.12.2.0
+  - semigroups-0.18.5
+  - temporary-1.2.1.1
+  - th-compat-0.1.2@sha256:3d55de1adc542c1a870c9ada90da2fbbe5f4e8bcd3eed545a55c3df9311b32a8,2854
+  - bytestring-encoding-0.1.0.0@sha256:460b49779fbf0112e8e2f1753c1ed9131eb18827600c298f4d6bb51c4e8c1c0d,1727
+  - hiedb-0.4.0.0
+  - dependent-map-0.4.0.0@sha256:ca2b131046f4340a1c35d138c5a003fe4a5be96b14efc26291ed35fd08c62221,1657
+  - dependent-sum-0.7.1.0@sha256:5599aa89637db434431b1dd3fa7c34bc3d565ee44f0519bfbc877be1927c2531,2068
+  - dependent-sum-template-0.1.0.3@sha256:0bbbacdfbd3abf2a15aaf0cf2c27e5bdd159b519441fec39e1e6f2f54424adde,1682
+  - constraints-extras-0.3.1.0
+  - some-1.0.1@sha256:26e5bab7276f48b25ea8660d3fd1166c0f20fd497dac879a40f408e23211f93e,2055
+  - unliftio-core-0.2.0.1@sha256:9b3e44ea9aacacbfc35b3b54015af450091916ac3618a41868ebf6546977659a,1082
+  - lsp-1.2.0.1
+  - lsp-types-1.3.0.0
+  - lsp-test-0.14.0.1
+
+  # Enable these when supported by all formatters
+  # - ghc-lib-9.0.1.20210324@sha256:c8b9a2541ea3424c8d0e4f80584477d0f35be03f4a47d931152042d5f446c5fc,19279
+  # - ghc-lib-parser-9.0.1.20210324@sha256:fb680f78d4ab08b5d089a05bda3b84ad857e5edcc2e4ca7c188c0207d369af80
+  # - ghc-lib-parser-ex-9.0.0.4@sha256:8282b11c3797fc8ba225b245e736cc9a0745d9c48d0f9fea7f9bffb5c9997709,3642
+  # - hlint-3.3@sha256:4218ad6e03050f5d68aeba0e025f5f05e366c8fd49657f2a19df04ee31b2bb23,4154
+
+configure-options:
+  ghcide:
+      - --disable-library-for-ghci
+  haskell-language-server:
+      - --disable-library-for-ghci
+  heapsize:
+      - --disable-library-for-ghci
+
+flags:
+  haskell-language-server:
+    pedantic: true
+  retrie:
+    BuildExecutable: false
+  # Stack doesn't support automatic flags.
+  # Until the formatters support ghc-lib-9, we need this flag disabled
+  hls-hlint-plugin:
+    hlint33: false
+  hyphenation:
+    embed: true
+
+nix:
+  packages: [ icu libcxx zlib ]
+
+concurrent-tests: false
```

Comment:
It looks like the GHC test-against list in the cabal file isn't updated?

Reply:
yeah, thanks for noting it